shared/charts/gpii-flowmanager/values.yaml

```diff
@@ -36,3 +36,13 @@ ingress:
   disable_ssl_redirect: false
 
 nameOverride: flowmanager
+
+## Optional resource requests and limits for deployment
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources: {}
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # limits:
+  #  cpu: 56
```

Comment:
This value looks strange.

Reply:
This is an example resource section I copied from some other chart. You don't think that our flowmanager deployment may require 56 CPUs and 256GB RAM one day? :)

Reply:
Ok. I didn't notice that the memory limit was GB.